src/csvcubedmodels/rdf/dcat.py

```diff
@@ -165,3 +219,24 @@ def __init__(self, uri: str):
         self.rdf_types.add(DCAT.Catalog)
 
         self.records = set()
+
+class Catalog(Distribution):
```

Comment:
Why have you chosen to include `Catalog` here? I don't believe we need it, do we?

It certainly shouldn't inherit from `Distribution`.